docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -275,3 +275,37 @@ Total Test time (real) =  83.61 sec
 ```
 
 A clean pass of the above unit tests is an indication that your Open Enclave setup was successful.
+
+## 6. (Optional) Verify OE remote attestation on pre-production platforms
+You will need to make some changes in Step 4.1 and 5 above when using the pre-production platforms.
+
+### 6.1 Get SBX (Sand Box) primary key for Provisioning Certificate Caching Service (PCCS) in Step 4.1
+Get the SBX primary key from https://sbx.api.portal.trustedservices.intel.com/provisioning-certification.
+The steps of installing and configuring PCCS are similar to Step 4.2.
+
+### 6.2 Replace the Intel root certificate's public key value and CA name before Step 5
+
+common/sgx/quote.c
+ // Public key of Intel's root certificate.
+ static const char* g_expected_root_certificate_key =
+     "-----BEGIN PUBLIC KEY-----\n"
+-    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEC6nEwMDIYZOj/iPWsCzaEKi71OiO\n"
+-    "SLRFhWGjbnBVJfVnkY4u3IjkDYYL0MxO4mqsyYjlBalTVYxFP2sJBK5zlA==\n"
++    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAET/oP/VYc2tbA+Y0wjIEoxbknonMy\n"
++    "yOjrE/a+QrVx1kZvU8ZE/8L/wQKCIOSaSWbPAvMuL7TTSbssuu0okDegLQ==\n"
+     "-----END PUBLIC KEY-----\n";
+
+common/sgx/tcbinfo.c
+ // Public key of Intel's root certificate.
+ static const char* _trusted_root_key_pem =
+     "-----BEGIN PUBLIC KEY-----\n"
+-    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEC6nEwMDIYZOj/iPWsCzaEKi71OiO\n"
+-    "SLRFhWGjbnBVJfVnkY4u3IjkDYYL0MxO4mqsyYjlBalTVYxFP2sJBK5zlA==\n"
++    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAET/oP/VYc2tbA+Y0wjIEoxbknonMy\n"
++    "yOjrE/a+QrVx1kZvU8ZE/8L/wQKCIOSaSWbPAvMuL7TTSbssuu0okDegLQ==\n"
+     "-----END PUBLIC KEY-----\n";
+
+host/sgx/sgxquoteprovider.c
+ // Define the name of CA
+-static uint8_t CRL_CA_PROCESSOR[] = "processor";
++static uint8_t CRL_CA_PROCESSOR[] = "platform";
```

Comment:
The formatting for this is unreadable in markdown format. Please enclose this block with the `` ```patch`` markdown so that it renders correctly. For example:

````
```patch
common/sgx/quote.c
 // Public key of Intel's root certificate.
 static const char* g_expected_root_certificate_key =
     "-----BEGIN PUBLIC KEY-----\n"
-    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEC6nEwMDIYZOj/iPWsCzaEKi71OiO\n"
-    "SLRFhWGjbnBVJfVnkY4u3IjkDYYL0MxO4mqsyYjlBalTVYxFP2sJBK5zlA==\n"
+    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAET/oP/VYc2tbA+Y0wjIEoxbknonMy\n"
+    "yOjrE/a+QrVx1kZvU8ZE/8L/wQKCIOSaSWbPAvMuL7TTSbssuu0okDegLQ==\n"
     "-----END PUBLIC KEY-----\n";

common/sgx/tcbinfo.c
 // Public key of Intel's root certificate.
 static const char* _trusted_root_key_pem =
     "-----BEGIN PUBLIC KEY-----\n"
-    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEC6nEwMDIYZOj/iPWsCzaEKi71OiO\n"
-    "SLRFhWGjbnBVJfVnkY4u3IjkDYYL0MxO4mqsyYjlBalTVYxFP2sJBK5zlA==\n"
+    "MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAET/oP/VYc2tbA+Y0wjIEoxbknonMy\n"
+    "yOjrE/a+QrVx1kZvU8ZE/8L/wQKCIOSaSWbPAvMuL7TTSbssuu0okDegLQ==\n"
     "-----END PUBLIC KEY-----\n";

host/sgx/sgxquoteprovider.c
 // Define the name of CA
-static uint8_t CRL_CA_PROCESSOR[] = "processor";
+static uint8_t CRL_CA_PROCESSOR[] = "platform";
```
````

Reply:
OK, I will add it.